producer/src/main/java/com/klungerbo/streams/pulsar/Main.java

```diff
@@ -17,12 +17,13 @@
      *
      * @param args commandline arguments.
      */
-    public static void main(final String[] args) {
+    public static void main(final String[] args) throws InterruptedException {
         final Logger logger = LoggerFactory.getLogger(Main.class);
 
         int tries = 100;
         int currentTry = 1;
         int secondsToSleep = 5;
+        Thread.sleep(30000);
```

Comment:
This is a very special case. Should we add a comment to explain why we have decided to sleep for a short amount of time?